package.json

```diff
@@ -28,19 +24,18 @@
     "react-dom": ">=0.14.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-eslint": "^7.1.1",
-    "babel-loader": "^6.3.2",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-preset-es2015": "^6.24.1",
-    "eslint": "^3.15.0",
-    "eslint-config-airbnb": "^14.1.0",
-    "eslint-loader": "^1.8.0",
-    "eslint-plugin-import": "^2.3.0",
-    "eslint-plugin-jsx-a11y": "^4.0.0",
-    "eslint-plugin-react": "^6.9.0",
-    "pre-commit": "^1.2.2",
```

Comment:
Why pre-commit was removed?

Reply:
A couple of reasons
1) It puts scripts in your `.git/hooks` folder which I find weird and securitywise also a bit critical
2) I think a CI is a better place to enforce code style, and that allows everyone to have his own local workflow while still enforcing style.
3) Sometimes it's annoying if you make lot of commits and then have to wait for the script to run even though everything is already linted.
I'd rather not have it, but since we don't have a CI on this project I will bring it back if you want.

Reply:
Or I can check out github actions :-)

Reply:
Ok, it's a matter of taste ... I just don't like pushing code with linting errors. Maybe pre-push instead of pre-commit wouldn't be that annoying. Or just setup CircleCI or try out actions.